docs/dataset/formatting-your-datasets.md

```diff
@@ -90,14 +98,63 @@ These annotations are visible on both the Gallery view for groups of datapoints
+| s3://kolena-public-examples/coco-2014-val/data/COCO_val2014_000000369763.jpg | trunk      | 313.02    | 553.98 | 12.01 | 99.84   |
+
+The first bounding box for the image is `(270.77, 44.59), (621.61,  254.18)`. To represent this within Kolena use the
+[`BoundingBox`](../reference/annotation.md#kolena.annotation.BoundingBox) annotation. This looks like:
+
+```python
+from kolena.annotation import BoundingBox
+bbox = BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61,  254.18))
+```
+When viewing a bounding box within python the format is
+`BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61, 254.18), width=350.84, height=209.59, area=73532.5556, aspect_ratio=1.67)`
+
+A single bounding box would be serialized as the following JSON string within a CSV:
+
+```
+{""top_left"": [270.77, 44.59], ""bottom_right"": [621.61, 254.18], ""width"": 350.84, ""height"": 209.59,
+ ""area"": 73532.5556, ""aspect_ratio"": 1.67, ""data_type"": ""ANNOTATION/BOUNDING_BOX""},
+```
+
+The above example has multiple objects within a single image, in order to ingest this you would need a single field
+containing the list of bounding boxes.
+
+For example:
+```python
+from kolena.annotation import BoundingBox
+bboxes = [
+    BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61, 254.18)),
+    BoundingBox(top_left=(538.03, 8.86), bottom_right=(636.85, 101.93)),
+    BoundingBox(top_left=(313.02, 12.01), bottom_right=(553.98, 99.84)),
+]
+```
+This would be represented within a csv as shown below. Note this will be a single line,
+but is shown here as multiple lines for formatting.
+```
+"[{""top_left"": [270.77, 44.59], ""bottom_right"": [621.61, 254.18], ""width"": 350.84, ""height"": 209.59,
+ ""area"": 73532.5556, ""aspect_ratio"": 1.67, ""data_type"": ""ANNOTATION/BOUNDING_BOX""},
+  {""top_left"": [538.03, 8.86], ""bottom_right"": [636.85, 101.93], ""width"": 98.82, ""height"": 93.07,
+   ""area"": 9197.1774, ""aspect_ratio"": 1.062, ""data_type"": ""ANNOTATION/BOUNDING_BOX""},
+  {""top_left"": [313.02, 12.01], ""bottom_right"": [553.98, 99.84], ""width"": 240.96,
+   ""height"": 87.83, ""area"": 21163.5168, ""aspect_ratio"": 2.743, ""data_type"": ""ANNOTATION/BOUNDING_BOX""}]"
+```
+
 When uploading `.csv` files for datasets that contain annotations, assets or nested values in a column use the
 [`dataframe_to_csv()`](../reference/io.md#kolena.io.dataframe_to_csv) function provided by Kolena to save a `.csv` file
-instead of [`pandas.to_csv()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html).
-
-In order to add data like a list of `BoundingBox` objects to your dataset via the sdk all you need to do is have
-field with a list of objects in your dataframe.
+instead of [`pandas.to_csv()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html). The above
+serializations were created using `dataframe_to_csv()`.
```

Comment:
Should probably have a brief blurb that `pandas.to_csv` won't properly serialize Kolena annotation objects. Right now we are just saying don't use it without an explanation.

nit: I think we can leave out: "The above serializations were created using `dataframe_to_csv()`."

Reply:
Updated